### DISCUSSION TITLE
embeddable widget | filter by dates

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3580] [Q&A] [answered]
Hello!
Is there an option to filter out specific dates from displaying in the calendar by using the embeddable widgets ?
For example, something like
pretix-widget event="https://pretix.eu/demo/democon/" items="<product_id>" date="16/10/2023"></pretix-widget

**Comment by wiffbi** [comment, marked as answer]
Not directly, but you could use the meta-data feature for e.g. dates and use that to filter events. See https://docs.pretix.eu/en/latest/user/events/widget.html#multi-event-selection

> You can filter events by meta data attributes. You can create those attributes in your order profile and set their values in both event and series date settings. For example, if you set up a meta data property called “Promoted” that you set to “Yes” on some events, you can pass a filter like this:
> 
> `<pretix-widget event="https://pretix.eu/demo/series/" list-type="list" filter="attr[Promoted]=Yes"></pretix-widget>`

**Reply by alemao8** [reply]
This worked. Thank you
